ivy/functional/frontends/tensorflow/nn.py

```diff
@@ -18,6 +18,11 @@ def _reduce_strides_dilations(dim, stride, dilations):
     return stride, dilations
 
 
+@to_ivy_arrays_and_back
+def embedding_lookup(params, ids, max_norm=None, name=None):
+    return ivy.embedding_lookup(params, ids)
```

Comment:
There is no `embedding_lookup` function in the Ivy API, you'll need to implement a composition for it, here is a guide https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html#temporary-compositions

Reply:
I reviewed your stated source and each and every source it pointed to. I found an example but there is no explanation how has that been done. 
Also, how to know if my chosen function is included in the Ivy API?

Reply:
If the function is not present in the ivy functional API we use the available ivy functions to implement its behaviour, and suggest if the function should be added to the functional API here https://github.com/unifyai/ivy/issues/3856